src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php

```diff
@@ -250,6 +250,16 @@ public function setPendingDeletion($isPendingDeletion): void
         }
     }
 
+    /**
+     * @internal Used by the form types
+     *
+     * @return bool
+     */
+    public function isPendingDeletion()
+    {
+        return \strlen($this->oldFileName) > 0 && $this->fileName === null;
```

Comment:
Does this need to be `\strlen`?

Reply:
opcache optimalisations

Reply:
Care to share the docs of article mentioning this? :)

Reply:
I found something related to this:

https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048

Reply:
already shared it multiple times already :) it is not the first time I'm including it

Reply:
Don't really think it makes a difference in this use case, but ok